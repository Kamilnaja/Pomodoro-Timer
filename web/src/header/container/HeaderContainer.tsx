import React from 'react';
import { HeaderComponent } from '../component/HeaderComponent';
import { HeaderContainerProps } from './HeaderContainerProps';

export default class HeaderContainer extends React.Component<HeaderContainerProps> {
  render() {
    return <HeaderComponent handleOpenModal={this.props.handleOpenModal} isLoggedIn={this.props.isLoggedIn} />;
  }
}
