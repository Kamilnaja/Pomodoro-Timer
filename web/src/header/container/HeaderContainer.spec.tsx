import { shallow } from 'enzyme';
import { shallowToJson } from 'enzyme-to-json';
import HeaderContainer from './HeaderContainer';

describe('LoginContainer', () => {
  const component = shallow(<HeaderContainer isLoggedIn={true} handleOpenModal={() => {}} />);

  beforeEach(() => {});

  it('Should match the snapshot', () => {
    expect(shallowToJson(component)).toMatchSnapshot();
  });
});
