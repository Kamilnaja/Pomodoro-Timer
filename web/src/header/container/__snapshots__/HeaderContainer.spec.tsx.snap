// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`LoginContainer Should match the snapshot 1`] = `
<HeaderComponent
  handleOpenModal={[Function]}
  isLoggedIn={true}
/>
`;
